ddp-workspace/projects/ddp-sdk/src/lib/services/authentication/auth0Adapter.service.ts

```diff
@@ -353,9 +356,11 @@ export class Auth0AdapterService implements OnDestroy {
     }
 
     private handleExpiredTemporarySession(): void {
-        this.log.logToCloud(`${this.LOG_SOURCE} expired temporary session`);
-        this.session.clear();
-        window.location.reload();
+        this.log.logToCloud(`${this.LOG_SOURCE} expired temporary session`).subscribe(() => {
+                this.session.clear();
+                window.location.reload();
```

Comment:
This is the other case that I worried about. a location.reload will not result in changing domains but (maybe) could result in interrupting logToCloud in flight request.